backend/spec/controllers/spree/admin/orders_controller_spec.rb

```diff
@@ -202,17 +202,28 @@
         spree_get :confirm, id: order.number
       end
 
-      context 'when incomplete' do
-        before { allow(order).to receive_messages :completed? => false }
+      context 'when in confirm' do
+        before { allow(order).to receive_messages completed?: false, confirm?: true }
+
+        it 'is renders the confirm page' do
```

Comment:
and the is here seems superfluous also. adding value with my nitpicks every day!